docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+- Size-reduced coredumps for C# programs have been added.
+
+
+### System (Base)
+
+#### New and changed features
+
+- The capi-base-utils API set has been expanded.
+  - New
+    - ures: Resource management module.
+
+#### Fixes
+
+- The capi-system-system-settings API has been refactored.
+
+
+### Application framework
+
+#### New and changed features
+
+- TIDL Extension
+  - API set
+    - The API set for synchronous connection has been added.
+    - New keywords for sharing files have been added.
+  - Tools
+    - Tools for analyzing latency and throughput have been provided.
+    - The enhanced TIDL compiler using receive threads to improve concurrency has been provided.
+- Multi-package installer
+  - API set
+    - The API set has been provided to install multiple packages at once.
+  - Tools
+    - Tools have been provided to install multiple packages at once.
+- Tizen Theme Manager
+  - Getting resources
+    - The API set for obtaining related resources path using a specific resource key has been provided.
+  - Theme changed event
+    - The API set for receiving the changed event from the app when changing the settings has been provided.
+  - Theme overlay
+    - Overlay function between default theme and selected theme has been provided.
+
+
+### Window and Interaction
+
+#### New and changed features
+
+- Interaction
+  - Gesture FW & engine
+    - The API set to receive the event for hand gesture has been supported.
+    - Gesture Framework for managing multi gesture clients and engines has been supported.
+    - The service app for default gesture engine has been supported.
+    - Support for on-demand daemon launch has been added.
+- Enlightenment Wayland display server
+  - Display Compatibility for the Legacy Widget Application has been added. Legacy Widget applications can display on the screen of which size is different from the one of application without the modification.
+- Ecore Buffer
+  - The ecore-buffer and ecore-buffer-queue API set has been deprecated.
+- Tizen-ws-shell
+  - A new enumeration has been added to support a new type of quickpanel service.
+- Quickpanel C# Support
+  - C# API set for quickpanel client and service has been added. (Tizen.NUI.WindowSystem.Shell)
+- The libds API set that can be used to create and configure wayland-based display server has been newly added.
+
+
+### Graphics Engine
+
+#### New and changed features
+
+- DALi (3D UI Toolkit)
+  - All API set of DALi has been deprecated. NUI (DALi C# layer) replaces DALi API set.
+- NUI
+  - Common
+    - Throwing an exception has been added when some methods are used in worker threads.
+    - Support for ComponentApplications has been added.
+  - Image
+    - Support for WebP image has been added.
+    - Support for asynchronous loading of gif images has been added.
+  - Input
+    - The keyboard repeat setting changed event has been added.
+  - Utility
+    - Some Capture API set has been added.
```

Comment:
@mijin85cho 
If it was like: 
- Some Capture API sets 
then 
have been added. 
is appropriate

But it is Some API set 

Since set is a collection but is singular in itself, thus has will satisfy the agreement in the context of a complete sentence. 

Some API set ( it is a set of x APIs) that has been added. 

- A set of X APIs has been added. 
- Some set of API has been added. 
- Some set of APIs have been added 

- Some Capture API set has been added. 
- Some Capture API sets have been added. 

What are your views? Let me know if I am wrong. 
Thanks.

Reply:
@nawab-samsung In my view, "some" itself has the meaning of plural.
So to be used with "some", the "set" must be changed to "sets", the plural form.

I asked this question because I thought "Some set" should be checked also. 
Isn't the "Some sets" the right form, resulting "have" as the following verb form?

I tried to check which "Capture API set" is/are added to check plural & singular, but couldn't. So I asked at here.

Reply:
What is correct w.r.t the context? 

Can you please let us know many API sets for Capture have been added? 
If it is one, we can explicitly write its name.

Reply:
As "API set" is being used instead of "APIs" in the release note, we'd like to avoid using "API sets". By the way how about using "APIs" instead of "API set" only in this sentence if "APIs" is commonly used (if not the wrong expression)?

Reply:
@wtprince  thanks, we do understand that an API set is added, which means (multiple APIs ) have been added. 

@mijin85cho we can go ahead with the same text as is then, what are your thoughts?

Reply:
@wtprince I understand that you don't want to use "API sets", so for here we can consider "API set" as "API sets", plural. For the consistency through the document, I hope to keep "API set" instead of "APIs".